Python/ceval.c

```diff
@@ -596,10 +605,12 @@ _Py_CheckRecursiveCall(PyThreadState *tstate, const char *where)
     assert(_tstate->c_stack_soft_limit != 0);
```

Comment:
The comment on `_Py_CheckRecursiveCall` is now outdated. Could it mention `_Py_MakeRecCheck`?